client/src/pages/CreateUserProfile.js

```diff
@@ -0,0 +1,219 @@
+    )
+};
+
+const StyledMenu = styled(Menu)`
+    padding: ${theme.typography.size.xxsmall};
+    width: 100%;
+    .ant-dropdown-menu-item {
+        display: flex;
+        span {
+            font-size: ${theme.typography.size.medium};
+            margin-right: ${theme.typography.size.xxsmall};
+            :first-child {
+                color: ${theme.colors.darkerGray};
+                font-weight: 500;
+            }
+            :last-child {
+                margin: 0;
+            }
+        }
+        &.ant-dropdown-menu-item-active {
+            background-color: ${theme.colors.selago};
+        }
+    }
+`;
+
+const DropdownMenu = ({children}) => {
+    const menu = (
+        <StyledMenu>
+            <Menu.Item>
+               <span>10014</span>
+               <span>13th Ave,</span>
+               <span>New York,</span>
+               <span>NY,</span>
+               <span>USA</span>
+            </Menu.Item>
+            <Menu.Divider/>
+        </StyledMenu>
+    );
+    return <Dropdown overlay={menu}>
+        {children}
+    </Dropdown>
+}
+
+const InputGroup = styled.div`
+    --py: ${theme.typography.size.xxsmall};
+    --my: ${theme.typography.size.xxxlarge};
+    
+    margin: var(--my) 0 var(--my) 0;
+
+    h3 {
+        color: ${theme.colors.primary};
+        font-weight: bold;
+    }
+    input {
+        color: ${theme.colors.darkerGray};
+        font-size: ${theme.typography.size.xlarge};
+        width: 100%;
+        margin-bottom: var(--my);
+        padding: var(--py) 0 var(--py) 0;
+
+    }
+    label {
+        font-family: ${theme.typography.font.family.button};
+        font-size: ${theme.typography.size.large};
+        font-weight: 500;
+    }
+`
+
+const UnderlineLink = styled.p`
+    --color: ${theme.colors.darkGray};
+    color: var(--color) !important;
+    a {
+        color: var(--color);
+        text-decoration: underline;
+    }
+    font-size: ${theme.typography.size.small};
+`;
+
+const CreateProfile = () => {
+return <Container>
+    <Flex className="image-container" direction="column">
+        <img src={PersonalDataImage} alt="two people standing next to a list of personal items" />
+    </Flex>
+    <Flex className="form-container" direction="column">
+        <WhiteSpace size="xl"/>
+        <ProfileFormGroup>
+            <CustomH1 color={theme.colors.darkerGray} fontweight="bold">Create your Profile</CustomH1>
+            <InputGroup>
+                <TextInput label="E-mail" />
+                <TextInput label="Name" />
+                <DropdownMenu>
+                    <div id="dropdown-anchor" style={{'position': 'relative'}}>
+                        <TextInput label="Address"/>
+                    </div>
+                </DropdownMenu>
+            </InputGroup>
+            <CheckboxGroup description="I am traveling" />
+            <CheckboxGroup description="Don't show my address" />
+            <InputGroup>
+                <Title>I want to</Title>
```

Comment:
Need to update the styling a bit:

current:
![Screen Shot 2020-05-07 at 2 36 10 PM](https://user-images.githubusercontent.com/42228386/81347302-b2e04300-9070-11ea-96c5-b61a5d8eb205.png)

figma:
![Screen Shot 2020-05-07 at 2 36 19 PM](https://user-images.githubusercontent.com/42228386/81347312-b83d8d80-9070-11ea-81b9-19a81a767708.png)
![Screen Shot 2020-05-07 at 2 36 27 PM](https://user-images.githubusercontent.com/42228386/81347318-bb387e00-9070-11ea-8ee4-fcea60b8a533.png)